app/Http/Controllers/CourseController.php

```diff
@@ -5,41 +5,58 @@
 namespace App\Http\Controllers;
 
 use App\Course;
+use App\Http\Requests\StoreCourse;
+use App\Http\Requests\UpdateCourse;
 use Illuminate\Http\JsonResponse;
-use Illuminate\Http\Request;
 use Illuminate\Http\Response;
 
 class CourseController extends Controller
 {
+    /**
+     * Show list of all Courses.
+     */
+    public function index(): JsonResponse
+    {
+        $courses = Course::all();
+
+        return response()->json($courses);
+    }
+
     /**
      * Store a newly created resource in storage.
      */
-    public function store(Request $request): JsonResponse
+    public function store(StoreCourse $request): JsonResponse
     {
-        return response()->json([], Response::HTTP_NOT_IMPLEMENTED);
+        Course::create($request->validated());
```

Comment:
Are these the same changes as in #5 pull request? You need to organize your work to avoid situations like this.

Reply:
I fail to see any changes to CourseController.php in #5